### PR TITLE
[devex] Redirect manually to / after logout when in dev env  (#385)

### PR DESCRIPTION
**Pull request by guillaumejparis**

### CODE REVIEW

File: openex-front/src/admin/components/nav/TopBar.js [hunk 1]
```diff
@@ -89,6 +89,10 @@ class TopBar extends Component {
   handleLogout() {
     this.handleClose();
     this.props.logout();
+    if (process.env.NODE_ENV === 'development') {
+      // api logout success handler redirect to / or in dev it is localhost:8080 and not localhost:3000
```
Comment (guillaumejparis) [inline openex-front/src/admin/components/nav/TopBar.js:93]:
```suggestion
      // api logout success handler redirect to / but in dev it is localhost:8080 and not localhost:3000
```

**Comment by RomuDeuxfois** [comment]
Works fine, but I don't find any mention of NODE_ENV=production for Docker image..
It's maybe missing ?

We need to be sure about this point before merging I think

**Comment by guillaumejparis** [comment]
> Works fine, but I don't find any mention of NODE_ENV=production for Docker image.. It's maybe missing ?
> 
> We need to be sure about this point before merging I think

We use it in the store.js file so I suppose it works but we can double check just to be sure yea !

**Comment by RomuDeuxfois** [comment]
Put the logic in backend

**Comment by guillaumejparis** [comment]
to do in backend

**Pull request closed without merging**